core/system/src/api.c

```diff
@@ -171,4 +173,6 @@ const UvisorApi __uvisor_api = {
         .thread_destroy = thread_destroy_transition,
         .thread_switch = thread_switch_transition,
     },
+
+    .debug_semihosting_enable = debug_semihosting_enable,
```

Comment:
So, just to clarify to myself: This function will only work if called in privileged mode or from secure code. This is fine, since it is supposed to be called by the debugger after reset. Nice design!